src/utils/api/btc/account.js

```diff
@@ -0,0 +1,42 @@
+import * as bitcoin from 'bitcoinjs-lib';
+import Lisk from 'lisk-elements';
+import bip32 from 'bip32';
+import * as popsicle from 'popsicle';
+import getBtcConfig from './config';
+
+export const getSummary = (address, netCode = 1) => new Promise(async (resolve, reject) => {
+  try {
+    const config = getBtcConfig(netCode);
+    const response = await popsicle.get(`${config.url}/balance?active=${address}`)
```

Comment:
This endpoint is different on the bridge -> https://github.com/LiskHQ/bitcoin-bridge#account-data-by-account-id

```suggestion
    const response = await popsicle.get(`${config.url}/account/${address}`)
```